podcast/2.12.m

```diff
@@ -0,0 +1,933 @@
+
+**Beyang Liu:**
+
+So you were quite far from the House of Cards power struggles.
+
+**Joseph Nelson:**
+
+So, back when I was passing the Bush tax cuts, yeah.
+
+No, honestly, I actually set up a Google alert and it did my job better than I did. So that was demoralizing. But there was one day where I worked as a legislative intern, and I was sorting mail, and it just stood out to me how inefficient the process was. And not for any fault of the offices. In fact, Grass's office is famous for doing a pretty good job of being really specific and responsive to constituents.
+
+Some offices will have just one mail they send to everyone. It's like, “Thanks for contacting us. We'll take your thing into consideration.” And then there's a sliding scale, a gradient of how specific you could be. You could actually address the issue that someone wrote in about, and then give the office's position back. And everyone who writes to their member of Congress should expect at least a response, and you'd hope for a pretty tailored response. But when you view it from the other side, from the institutional side, and you're so understaffed and there are such high volumes, you can understand why you get somewhat form-letter-type responses. Absent that, there's just no response possible.
+
+And that's how I was first exposed to the problem, but it wasn't until years later, when I picked up a little bit more programming, and by then I was actually working at Facebook in 2015. It was my last year of school but I didn't really want to be done with school, so I basically took one class at night and then worked full-time in the Facebook DC office. So, there's 50 people there. It was pretty cool. And Facebook hosts, or used to host, the annual Facebook Congressional Hackathon. If we could just take a minute to think about if they still called it that and had that today, how-
+
+**Beyang Liu:**
+
+Hacked and congressional in the same sentence would be...
+
+**Joseph Nelson:**
+
+And Facebook, that's already-
+
+**Beyang Liu:**
+
+And Facebook.
+
+**Joseph Nelson:**
+
+But anyway, so, the annual Facebook congressional hackathon, and by annual, I think it happened twice every other year. And I was re-confronted with this problem, but the difference was that offices showed up and came to say things about the types of issues they were facing and solutions they were hoping to see to their problems. And I was re-confronted with the same thing that I felt as an intern, except this time I felt like I had some level of programming skills to change that.
+
+So, we hacked together. I don't know if you know the open source library spaCy–it's an NLP library. It's pretty good. Burton would probably outpace it today, but at the time... I guess you can use Burton spaCy, but anyway. SpaCy was pretty good for building an off-the-shelf topic model and tokenizing text in an efficient way for identifying when messages were about similar content.
```

Comment:
I haven't checked the audio recording, but I'm assuming that "Burton" is actual a reference to the BERT family of NLP models. "Burton spaCy" might be BERT in spaCy (which is actually possible these days).